pkg/hwevent/event_data.go

```diff
@@ -89,44 +89,45 @@ type EventRecord struct {
 // String returns a pretty-printed representation of the EventRecord.
 func (e EventRecord) String() string {
 	b := strings.Builder{}
+	b.WriteString("\n")
 	if e.Actions != nil {
-		b.WriteString("	Actions: " + string(e.Actions) + "\n")
+		b.WriteString("      Actions: " + string(e.Actions) + "\n")
```

Comment:
you have a tab here, can you make it a single space? why you used tabs

Reply:
It is not a tab. It is 6 spaces. The reason I use it is to make the who event message looks better since the `EventRecord` is a few levels down. The following is what the event looks like when I print it out:
```
DEBU[0019] published hw event id: 79cc779f-2587-492a-bcb6-b12e0f35a088
type: HW_EVENT
time: 2021-07-21T21:00:57.019946713Z
data:
  version: v1
  data:
    @odata.context: /redfish/v1/$metadata#Event.Event
    @odata.type: #Event.v1_3_0.Event
    Context: any string is valid
    Id: c8a5433e-de17-11eb-8d4b-f40270e4f7f4
    Name: Event Array
    Events[0]:
      Context: any string is valid
      EventGroupId: 0
      EventId: 2162
      EventTimestamp: 2021-07-06T01:17:12-0400
      EventType: Alert
      MemberId: 79694
      Message: The system inlet temperature is greater than the upper warning threshold.
      MessageId: TMP0120
      Severity: Warning 
```